db/schema.rb

```diff
@@ -20,17 +20,17 @@
     t.string "text"
```

Comment:
There is an error while migrating the database tables. So I was unable to run the project and test it. Kindly make sure there is no database error.